.github/workflows/_open_pr_with_new_snapshots.yml

```diff
@@ -0,0 +1,101 @@
+---
+name: Open PR with new snapshots
+
+on:
+  workflow_call:
+    inputs:
+      app_repository:
+        description: 'The GIT repository to test (defaults to `github.repository`)'
+        required: false
+        default: ${{ github.repository }}
+        type: string
+      app_branch_name:
+        description: 'The GIT branch to test (defaults to `github.ref`)'
+        required: false
+        default: ${{ github.ref }}
+        type: string
+      snapshots_artifact_name:
+        description: 'The artifact name'
+        required: true
+        type: string
+      snapshots_directory:
+        description: 'The directory in which the snapshots are extracted'
+        required: true
+        type: string
+
+jobs:
+  commit_new_snapshots:
+    name: Commit new snapshots
+    runs-on: ubuntu-latest
+    steps:
+      - name: Refuse to run on anything else than workflow dispatch
+        if: ${{ github.event_name != 'workflow_dispatch' }}
+        # Running this job on PRs or push could get out of hand
+        run: |
+          echo "This workflow is only allowed to run on 'workflow_dispatch' events."
+          exit 1
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ inputs.app_repository }}
+          ref: ${{ inputs.app_branch_name }}
+          submodules: recursive
+          # by default the action uses fetch-depth = 1, which creates
+          # shallow repositories from which we can't push
+          fetch-depth: 0
+          # needed, else the push inside the action will use default credentials
+          # instead of provided ones
+          persist-credentials: false
+
+      - name: Download new snapshots
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ inputs.snapshots_artifact_name }}
+          path: ${{ inputs.snapshots_directory }}/snapshots
```

Comment:
suffix /snapshots is already given in `reusable_ragger_tests.yml`?

Reply:
Nice catch, that was a last minute change that I did not test